server/src/main/java/com/google/musicanalysis/types/NameAndChannel.java

```diff
@@ -0,0 +1,11 @@
+package com.google.musicanalysis.types;
+
+public class NameAndChannel {
```

Comment:
Consider renaming this "VideoInfo", since this class might not always hold just the name and channel of a video.